ButtonStateless/index.jsx

```diff
@@ -102,6 +104,15 @@ const Button = ({
       color: outerSpace,
       backgroundColor: transparent,
     },
+    quaternary: {
```

Comment:
Splitting hairs here! If we add another button color/background I think we should allow for setting things like `color` and `background` in normal and hovered states.